Frameworks/BlockchainCertificates/Certificate.swift

```diff
@@ -227,6 +242,8 @@ private enum MethodsForV1_1 {
         }
         
         var hashed : Bool = false
+        let name : String = givenName + " " + familyName
```

Comment:
Oh, and it can infer the type from that.  `let name = "\(givenName) \(familyName)"`